src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const firstName = document.getElementById('sign-up-name');
+const lastName = document.getElementById('sign-up-last-name');
+const email = document.querySelectorAll('input[name="email"]');
+const password = document.querySelectorAll('input[name="password"]');
```

Comment:
```suggestion
const inputs = document.querySelectorAll('.field-text');
```
try to make more flexible solution.
select all inputs and use for example forEach method to fix each input, instead of hardcode and repeat code for each input.
```
inputs.forEach(input => {
   // do something
})